firefox-ios/firefox-ios-tests/Tests/AccessibilityTestPlan.xctestplan

```diff
@@ -392,6 +392,7 @@
         "TabCounterTests",
         "TabCounterTests\/testTabDecrement()",
         "TabCounterTests\/testTabIncrement()",
+        "TabsTests",
```

Comment:
`TopTabsTests` wasn't excluded from AccessibilityTestPlan. It should not be run here.